samples/sas_resource_files/README.md

```diff
@@ -0,0 +1,11 @@
+# SAS Resource Files
+
+The following sample show show to transfer data using secure [SAS blob tokens](https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1). This allows secure transfer to and from cloud storage from your local computer or the nodes in the cluster.
```

Comment:
typo? "show show" --> "shows how"?
"to and from cloud storage from" seems inconsistent, how about "between ... and"?